jobfunnel/jobfunnel.py

```diff
@@ -36,30 +38,25 @@ def __init__(self, args):
         # search term configuration data
         self.search_terms = args['search_terms']
 
-        # create dirs
+        # create data dir
         if not os.path.exists(args['data_path']): os.makedirs(args['data_path'])
 
-        # handle Python 2 & 3 Unicode formatting
-        try:
-            self.encoding = unicode
-        except NameError: # @TODO use sys.version_info instead
-            self.encoding = str
-
```

Comment:
this ends python2 support

Reply:
If this ends Python2 support then be sure to add the issue number in the PR description so it can be closed 😃